docs/int/openapi.yaml

```diff
@@ -1106,7 +1296,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/JobCode'
-
+    MetaDataPatch:
+      type: array
+      items:
+        $ref: '#/components/schemas/JSONPatchRequestReplace'
```

Comment:
Please add a line break between components per the current specfile's formatting.